Vinilos/app/src/main/java/com/tsdc/vinilos/repository/album/AlbumRepositoryImpl.kt

```diff
@@ -31,6 +33,16 @@ class AlbumRepositoryImpl(
         return datasourceLocal.getAlbums()
     }
 
+    override suspend fun createAlbum(album: JsonObject){
+        try {
+            val albumSaved = dataSourceRemote.createAlbum(album)
+            datasourceLocal.createAlbum(albumSaved)
+            Log.e("SaveAlbum", "Album saved successfully in both remote and local databases.")
```

Comment:
Esto no es correcto porque Log.e indica que es un error pero el comentario dice que es algo success, si deseas dejarlo, puedes colocarlo con Log.d
```suggestion
            Log.d("SaveAlbum", "Album saved successfully in both remote and local databases.")
```

Reply:
Solucionado